config.json

```diff
@@ -4,7 +4,10 @@
   "repository": "https://github.com/exercism/xkotlin",
   "active": true,
   "deprecated": [
-
+    "accumulate",
```

Comment:
If you have time that would be great, otherwise I can try to get to it tomorrow.